### DISCUSSION TITLE
Feature Request: Clear App Data just like cache using accesibility service

### DISCUSSION THREAD

**Discussion by tathastu871** [d4rken-org/sdmaid-se#357] [Ideas]
Been a long term user with unlocker for pro.
Just like sdmaid clears cache by clicking Clear cache button in settings using accesibility

If its possible add feature where sdmaid clicks clear data button on non root devices

Sometimes i had need to reset all apps on my device by clearing data without formatting phone itself.

I have already requested this feature to dev via email in past but none progress have been made.

**Comment by d4rken** [comment]
While this is possible, this seems like a something you rarely need to do.

Why do you need to reset multiple apps regularly that such automation would be useful?

I also reset apps occasionally, but it is always a single app that has misbehaved in that moment, so it's easier to open the system settings directly than user another app for it.

**Comment by tathastu871** [comment]
It have rare use. But for me its frequent. I had to reset all apps monthly. So it would be useful to have such feature.

**Comment by Wjxfi** [comment]
Cool idea!
